test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Culture.Tests.ps1

```diff
@@ -1,25 +1,68 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Get-Culture DRT Unit Tests" -Tags "CI" {
-    It "Should works proper with get-culture" {
-        $results = get-Culture
-        $results -is "System.Globalization.CultureInfo" | Should -BeTrue
-        $results[0].Name | Should -Be $PSCulture
-    }
-}
 
 Describe "Get-Culture" -Tags "CI" {
 
     It "Should return a type of CultureInfo for Get-Culture cmdlet" {
 
-	Get-Culture | Should -BeOfType CultureInfo
+        Get-Culture | Should -BeOfType [CultureInfo]
 
     }
 
     It "Should have $ culture variable be equivalent to (Get-Culture).Name" {
 
-	(Get-Culture).Name | Should -Be $PsCulture
+        (Get-Culture).Name | Should -Be $PsCulture
+
+    }
+
+    It "Should return the specified culture with '-Name' parameter" {
+
+        $ci = Get-Culture -Name ru-RU
+        $ci | Should -BeOfType [CultureInfo]
+        $ci.Name | Should -BeExactly "ru-RU"
+    }
+
+    It "Should return specified cultures with '-Name' parameter" {
 
+        $ciArray = Get-Culture "", "ru-RU"
+        $ciArray | Should -BeOfType [CultureInfo]
+        ,$ciArray | Should -BeOfType [System.Array]
```

Comment:
prefixing this with a comma always make this an array, so this line doesn't really test anything related to this cmdlet.  I think line 30 is sufficient to validate you got an array